sandbox/Sandbox/Generated.cs

```diff
@@ -989,22 +988,21 @@ public void Serialize(ref MessagePackWriter writer, global::Abcdefg.Efcdigjl.Ate
 
 namespace MessagePack.Formatters
 {
-    using System;
-    using System.Buffers;
-    using MessagePack;
+    using global::System.Buffers;
+    using global::MessagePack;
 
     public sealed class ArrayTestTestFormatter : global::MessagePack.Formatters.IMessagePackFormatter<global::ArrayTestTest>
     {
 
-        public void Serialize(ref MessagePackWriter writer, global::ArrayTestTest value, global::MessagePack.MessagePackSerializerOptions options)
+        public void Serialize(ref global::MessagePack.MessagePackWriter writer, global::ArrayTestTest value, global::MessagePack.MessagePackSerializerOptions options)
         {
             if (value == null)
             {
                 writer.WriteNil();
                 return;
             }
 
-            IFormatterResolver formatterResolver = options.Resolver;
+            var formatterResolver = options.Resolver;
```

Comment:
Why? Please stop mixing irrelevant changes with your PRs. It just makes reviews take longer.
Also, even if you made this style change in a separate PR, I would reject it. `var` should only be used when the type name already appears elsewhere on the line such that it is obvious.

Reply:
> Please stop mixing irrelevant changes with your PRs.

I'm sorry. I take your pointing out seriously and will not do it next time.

> `var` should only be used when the type name already appears elsewhere on the line shuch that it is obvious.

I have a different thought that all generated C# codes must not use `using namespace` directives.
`using namespace` directive causes ambiguous type name errors.
`var` is one good solution to avoid `using namespace` directives like `global::`.
`var` and `global::` make the program more resilient.

Reply:
Oh! A very interesting point. Good insight into the danger of being explicit on typing in a type generated file. Ok, go ahead and use `var` for `IFormatterResolver`. But when you can use a keyword (e.g. `int`) please use that instead of `var`, as that should still be safe to use.